### ISSUE TITLE
There is no folder name "tmp/data_dhm..."

### ISSUE THREAD

**Issue by kjh224633** [xiaobaishu0097/ECCV-VN#13]
HI. I have an error like the bottom while I execute  "python main.py --model GraphModel --worker 12 --gpu-ids 0 1 --title a3c_graph" . It would be so thankful to give me a solution. THANK YOU !!!

There is no folder name "tmp/data_dhm..." Do I need to download some other datasets?  I already downloaded 3 folders that ordered in ReadMe. 
### error 
OSError: Unable to open file (unable to open file: name = '/tmp/data_dhm/AI2thor_Dataset/Scene_Data/FloorPlan414/depth.hdf5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
Process Process-11:
###

**Comment by keeper1314** [comment]
Have you solved this problem?

**Comment by jinHwanKim33** [comment]
Yes. I solved the problem. The problem was data directory PATH. You need to change the directory from authors's PC directory to your directory. Maybe "tmp/data_dhm" was author's local PC directory.
